projects/driver_controls/inc/ads1015_bitmasks.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+// ADS1015 I2C address definition (Datasheet Section 8.5.1.1)
+#define ADS1015_I2C_ADDRESS             0x48
+
+// ADS1015 I2C reset byte (Datasheet Section 8.5.1.2)
+#define ADS1015_RESET_BYTE              0x06
+
+// Used Reference voltage in mv.
+#define ADS1015_REFERENCE_VOLTAGE       3300
```

Comment:
This should probably not be hardcoded - if anything, it's more around 2978mV or something. We always use an LDO for our analog circuits.